Python/Product/PythonTools/PythonTools/Options/PythonDebuggingOptionsControl.cs

```diff
@@ -13,13 +13,24 @@
 //
 // See the Apache Version 2.0 License for specific language governing
 // permissions and limitations under the License.
-
+using Microsoft.PythonTools.Debugger;
+using System;
+using System.Collections.Generic;
 using System.Windows.Forms;
 
 namespace Microsoft.PythonTools.Options {
     public partial class PythonDebuggingOptionsControl : UserControl {
         public PythonDebuggingOptionsControl() {
             InitializeComponent();
+
+            // set allowed values for the variable presentation options
```

Comment:
I think you can simplify this even further if you just set `DataSource` of every combo box directly to the array returned by `Enum.GetValues()`.

Reply:
Thank you Pavel, I will give that a shot 😄

Reply:
Works like a charm